packages/odyssey-react-mui/src/properties/ts/odyssey-react-mui.ts

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable header/header */
+export const translation = {"field.error.screen.reader.text":"Error","field.label.optional.text":"Optional","severity.success":"success","severity.warning":"warning","severity.info":"info","severity.error":"error","toast.close.text":"close"};
```

Comment:
Are we committing converted `.properties` translations to the repo?

I don't mind it, just curious. When they add `.properties` files, I'd like to generate these TS files in that step. Is that how you've set it up?